test/integration/content-preview/Annotations.e2e.test.js

```diff
@@ -0,0 +1,89 @@
+import('https://unpkg.com/box-annotations/dist/annotations.js');
```

Comment:
Yeah, e.g. "https://unpkg.com/box-annotations@4.0.0-beta.23/dist/annotations.js". However, I don't think we need to support this until we do hash verification in the future, since we always want to use the latest version.